File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/Icd10CodeEstimateLookup.cs

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class Icd10CodeEstimateLookup
    {
        public int Icd10CodeEstimateLookupId { get; set; }
        public int Icd10GroupMapId { get; set; }
        public int? ProductOptionId { get; set; }
        public int? MedicalCostLookupId { get; set; }
        public int? PdExtentLookupId { get; set; }
        public int? DaysOffLookupId { get; set; }
        public System.DateTime StartDate { get; set; }
        public System.DateTime EndDate { get; set; }
        public bool IsActive { get; set; }
        public bool IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }

        public virtual DaysOffLookup DaysOffLookup { get; set; }

        public virtual MedicalCostLookup MedicalCostLookup { get; set; }

        public virtual PdExtentLookup PdExtentLookup { get; set; }


    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/icd10-code-estimate-lookup.ts
import { DaysOffLookup } from './days-off-lookup';
import { MedicalCostLookup } from './medical-cost-lookup';
import { PdExtentLookup } from './pd-extent-lookup';

export interface Icd10CodeEstimateLookup {
    icd10CodeEstimateLookupId: number;
    icd10GroupMapId: number;
    productOptionId: number | null;
    medicalCostLookupId: number | null;
    pdExtentLookupId: number | null;
    daysOffLookupId: number | null;
    startDate: Date;
    endDate: Date;
    isActive: boolean;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;

    daysOffLookup: DaysOffLookup;
    medicalCostLookup: MedicalCostLookup;
    pdExtentLookup: PdExtentLookup;
}
